packages/types/src/tutor.ts

```diff
@@ -91,7 +91,7 @@ export type FindTutorInfoApiResponse = {
 
 export type FindOnboardedTutorsApiResponse = {
   total: number;
-  list: Array<Cache & { rules: IRule.RuleEvent[] }>;
+  list: Array<Cache & { slots: IRule.RuleEvent[] }>; // TODO: user AvailablitySlot.Self
```

Comment:
The frontend doesn't make use of `rules`. It only uses the data available in the `Cache`. We can safly remove it. 


```suggestion
  list: Array<Cache>; 
```